examples/multisig/MULTISIG_01.sol

```diff
@@ -1,14 +1,16 @@
 pragma solidity ^0.8.0;
 
-import {MultisigProposal} from "@proposals/MultisigProposal.sol";
-import {Addresses} from "@addresses/Addresses.sol";
 import {Vault} from "@examples/Vault.sol";
+import {Proposal} from "@proposals/Proposal.sol";
 import {MockToken} from "@examples/MockToken.sol";
+import {Addresses} from "@addresses/Addresses.sol";
+import {AlphaProposal} from "@proposals/AlphaProposal.sol";
+import {MultisigProposal} from "@proposals/MultisigProposal.sol";
 
 // MULTISIG_01 proposal deploys a Vault contract and an ERC20 token contract
 // Then the proposal transfers ownership of both Vault and ERC20 to the multisig address
 // Finally the proposal whitelist the ERC20 token in the Vault contract
-contract MULTISIG_01 is MultisigProposal {
+contract MULTISIG_01 is AlphaProposal, MultisigProposal {
```

Comment:
`buildModifier` not used for this proposal. If we are keeping this as a simple proposal then we can avoid inheriting `AlphaProposal` contract here.